tests/views/test_users.py

```diff
@@ -0,0 +1,22 @@
+import json
+import unittest
+
+import mock
+
+import crane.app
+
+
+class TestUsers(unittest.TestCase):
+    def setUp(self):
+        with mock.patch('crane.app.init_logging'):
+            self.app = crane.app.create_app().test_client()
+
+    def test_response(self):
```

Comment:
We also need a test for the case where it's a GET request. Travis failed because that code path isn't covered.